gateway/utils/graphql/read.go

```diff
@@ -20,12 +20,29 @@ func (graph *Module) execLinkedReadRequest(ctx context.Context, field *ast.Field
 		return
 	}
 
+	req.Aggregate, err = extractAggregate(field)
+	if err != nil {
+		cb("", "", nil, err)
+		return
+	}
+	var hasOptions bool
+	req.Options, hasOptions, err = generateOptions(field.Arguments, store)
+	if err != nil {
+		cb("", "", nil, err)
+		return
+	}
+	// if distinct option has been set then set operation to distinct from all
+	if hasOptions && req.Options.Distinct != nil {
+		req.Operation = utils.Distinct
+	}
```

Comment:
@YourTechBud should we support options for linked requests ?

Reply:
@YourTechBud ?

Reply:
No. We shouldn't